src/content/docs/ai-monitoring/explore-ai-data/view-model-data.mdx

```diff
@@ -0,0 +1,171 @@
+</figcaption>
+
+## Model inventory page [#model-inventory]
+
+<img
+    title="Model inventory overview"
+    alt="A screenshot of the overview page when you go to Model inventory"
+    src={aiModelInventoryOverviewPage}
+/>
+
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory</DoNotTranslate>**: View data about interactions with your AI model.
+</figcaption>
+
+The <DoNotTranslate>model inventory</DoNotTranslate> page provides insight into the overall performance and usage of your AI models. You can analyze data from calls made to your model, so you can understand how AI models affect your AI app.
+
+From the overview tab, explore the number of requests made to a model against its response times, or analyze time series graphs to see when model behavior changed. From there, investigate the errors, performance, or cost tabs.
+
+### Errors tab [#errors-inventory]
+
+<img
+    title="Model inventory: Errors"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryErrors}
+/>
+
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Errors</DoNotTranslate>**: View data about AI model errors.
+</figcaption>
+
+The errors tab uses time series graphs and tables to organize model errors. 
+
+* <DoNotTranslate>**Response errors**</DoNotTranslate>: Track the number of errors in aggregate that come from your AI model.
+* <DoNotTranslate>**Response errors by model**</DoNotTranslate>: Determine if one specific model produces more errors on average, or if one specific error is occurring across your models.
+* <DoNotTranslate>**Response errors by type**</DoNotTranslate>: View how often certain errors appear. 
+* <DoNotTranslate>**Errors table**</DoNotTranslate>: View error type and message in context of the request and response. 
+
+### Performance tab [#performance-inventory]
+
+<img
+    title="Model inventory: Performance"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryPerformancePage}
+/>
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Performance</DoNotTranslate>**: View data about your AI model's performance.
+</figcaption>
+
+The performance tab aggregates response and request metrics across all your models. Overview the models that take the most time to process a request or create a response with the pie charts, or refer to the time series graphs to track upticks in request or response times. 
+
+* The 95th percentile, or **p95**, indicates the number of transactions that were below a certain value during a time period. If you've filtered your time series to p95 and see that a call at 9:00pm took six seconds to complete, it means that 95% of all calls were below six seconds, but 5% of calls were above six seconds.
```

Comment:
As a user, I'm not quite sure I understand this. Why is it set to 6 seconds? Where do you set that number? The number 6 must be written in numeral according to our [User Guide](https://docs.newrelic.com/docs/style-guide/ui-writing/numbers-dates-and-times/).